File: SSIS/SSIS/Models/Core Models/RequestItem.cs
using System.ComponentModel.DataAnnotations;

namespace SSIS.Models
{
    public class RequestItem
    {
        [Key]
        public int RequestItemId { get; set; }
        public Item Item { get; set; }
        [Range(1, 1000)]
        public int Quantity { get; set; }
    }
}

File: Android/app/src/main/java/com/arun/api/Model/RequestItem.java
package com.arun.api.Model;

import java.io.Serializable;

public class RequestItem implements Serializable {
    int RequestItemId;
    Item Item;
    int Quantity;

    public int getRequestItemId() {
        return RequestItemId;
    }

    public void setRequestItemId(int requestItemId) {
        RequestItemId = requestItemId;
    }

    public com.arun.api.Model.Item getItem() {
        return Item;
    }

    public void setItem(com.arun.api.Model.Item item) {
        Item = item;
    }

    public int getQuantity() {
        return Quantity;
    }

    public void setQuantity(int quantity) {
        Quantity = quantity;
    }


}
